### PR TITLE
feat: bb logup proving support

### PR DESCRIPTION
**Pull request by Maddiaa0**
powdr side of https://github.com/AztecProtocol/aztec-packages/pull/5577

### CODE REVIEW

File: bberg/src/flavor_builder.rs [hunk 1]
```diff
@@ -35,7 +38,15 @@ impl FlavorBuilder for BBFiles {
         all_cols_and_shifts: &[String],
     ) {
         // TODO: move elsewhere and rename
```
Comment (jeanmon) [inline bberg/src/flavor_builder.rs]: Wouldn't it be the time to address or remove this TODO?

File: bberg/src/verifier_builder.rs [hunk 2]
```diff
@@ -4,21 +4,23 @@ use crate::{
...
-    fn create_verifier_cpp(&mut self, name: &str, witness: &[String]) {
+    fn create_verifier_cpp(&mut self, name: &str, witness: &[String], inverses: &[String]) {
         let include_str = includes_cpp(&snake_case(name));
 
         let wire_transformation = |n: &String| {
             format!(
             "commitments.{n} = transcript->template receive_from_prover<Commitment>(commitment_labels.{n});"
         )
         };
+        // TODO: make sure the inverse commitments are not included
```
Comment (jeanmon) [inline bberg/src/verifier_builder.rs]: Should this TODO also be handled in this PR?

**Review by jeanmon — APPROVED** [review]
LGTM. Please have a look at my minor comments.

File: bberg/src/verifier_builder.rs (hunk 2 above)
Comment (Maddiaa0) [reply bberg/src/verifier_builder.rs]: Ah it is handled, i can remove the todo

File: bberg/src/flavor_builder.rs (hunk 1 above)
Comment (Maddiaa0) [reply bberg/src/flavor_builder.rs]: will address

**Pull request merged**